lib/auth/in_memory/backend.js

```diff
@@ -139,6 +143,77 @@ const backend = {
         };
         return cb(null, vaultReturnObject);
     },
+
+    /**
+     * Mocks Vault's response to a policy evaluation request
+     * Since policies not actually implemented in memory backend,
+     * we allow users to proceed with request.
+     * @param {object} requestContextParams - parameters needed to construct
+     * requestContext in Vault
+     * @param {object} requestContextParams.constantParams -
+     * params that have the
+     * same value for each requestContext to be constructed in Vault
+     * @param {object} requestContextParams.paramaterize - params that have
+     * arrays as values since a requestContext needs to be constructed with
+     * each option in Vault
+     * @param {string} userArn - arn of requesting user
+     * @param {object} log - log object
+     * @param {function} cb - callback with either error or an array
+     * of authorization results
+     * @returns {callback} with either error or array of authorization results
```

Comment:
Technically, this returns `undefined`. Also, please check the syntax for callback definition in JSDoc.

http://usejsdoc.org/tags-callback.html